notebooks/Image_Analysis.ipynb

```diff
@@ -76,37 +76,96 @@
     "### Calcluating variance according to bbox"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cfb2a2dd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_bg_fg_vars(img_ids):\n",
+    "    bg_vars, fg_vars, full_img_var = [], [], []\n",
+    "    for img_id in tqdm(img_ids):\n",
+    "        img_meta_data = coco.loadImgs(ids=[img_id])\n",
+    "        file_name = img_meta_data[0]['file_name']\n",
+    "\n",
+    "        img_path = os.path.join(img_dir_path, file_name)\n",
+    "        laplacian = laplacian_img(img_path)\n",
+    "        full_img_var.append(laplacian)\n",
+    "\n",
+    "        ann_ids = coco.getAnnIds(imgIds=img_id)\n",
+    "        annotations = coco.loadAnns(ann_ids)\n",
+    "        bbox_var_lap, img_var_laplacian = get_bbox_fg_bg_var_laplacian(laplacian, annotations)\n",
+    "\n",
+    "        bg_vars.append(img_var_laplacian)\n",
+    "        fg_vars.extend(bbox_var_lap)\n",
+    "        \n",
+    "        return bg_vars, fg_vars, full_img_var\n",
+    "\n"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "28b2c9b5",
    "metadata": {},
    "outputs": [],
    "source": [
-    "bg_vars, fg_vars = [], []"
+    "bg_vars, fg_vars, full_img_var = get_bg_fg_vars(img_ids)\n",
```

Comment:
The outputs here are not consistent with the rest of the notebook. ```bg_vars_a``` and ```bg_vars_b``` are being used in some parts of the notebook and in some parts ```bg_vars``` is used. It is better to sanity check the whole notebook for compatibility.

Reply:
I'll do a sanity check. Thanks for pointing it out.